services/github/github-common-fetch.js

```diff
@@ -1,8 +1,9 @@
 'use strict'
 
```

Comment:
Would it make sense to move the GitHub release specific content into a separate helper module? I know there was previously domain-specific artifacts in here for Issues and a bit for Releases, but it's getting pretty Release heavy and I wonder if splitting would be helpful.

Reply:
Possibly, the release stuff seems to be exclusively used by the 3 services I refactored (`commits-since`, `release`, and `tag`), but I don't have enough context on this codebase to really know 🤷‍♂ I can do whichever.

I thought the audience for question was other collaborators

Reply:
> I thought the audience for this question was other maintainers

Sorry I could've been more clear, but I intended it as an open question for anyone, maintainer, author, or otherwise 😄 

> Possibly, the release stuff seems to be exclusively used by the 3 services I refactored

That was my sense as well and why I posed the question. There's a mix of other existing GitHub-related services, plus more that are planned/requested, that will utilize the non-release specific content from `github-common-fetch`. 

I'll leave it up to you, we don't need to block this on splitting them out, but if you feel like doing it that'd be great!

Reply:
Ok, I've split it off into `github-common-release` now. Think that's the last remaining comment here